tests/test_cfgfast.py

```diff
@@ -135,6 +135,18 @@ def test_busybox():
         yield cfg_fast_edges_check, arch, filename, edges_
 
 
+@attr(speed='slow')
+def test_ntoskrnl():
+    path = "C:\\Windows\\System32\\ntoskrnl.exe"  # we cannot distribute this file. as a result, this test case is manual
+    if not os.path.isfile(path):
+        raise nose.SkipTest("ntoskrnl.exe does not exist on this system.")
```

Comment:
I believe we can make this not specific to nose by using `@unittest.skipUnless`.

Reply:
Fixed!